packages/@aws-cdk/integ-tests-alpha/test/assertions/providers/integ.http-api-call-assertions.ts

```diff
@@ -0,0 +1,140 @@
+import { App, Stack } from 'aws-cdk-lib';
+import { ExpectedResult, IntegTest } from '../../../lib';
+import * as apigwv2 from 'aws-cdk-lib/aws-apigatewayv2';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as integrations from 'aws-cdk-lib/aws-apigatewayv2-integrations';
+
+const app = new App();
+const stack = new Stack(app, 'InvokeFunctionAssertions');
+const integ = new IntegTest(app, 'AssertionsTest', {
+  testCases: [stack],
+});
+
+const httpApi = new apigwv2.HttpApi(stack, 'HttpApi');
+
+httpApi.addRoutes({
+  path: '/get',
+  methods: [apigwv2.HttpMethod.GET],
+  integration: new integrations.HttpLambdaIntegration('GetIntegration',
+    new lambda.Function(stack, 'GetHandler', {
+      runtime: lambda.Runtime.NODEJS_18_X,
```

Comment:
Small nit to minimize our maintenance: Could you please make this node20 or LATEST? We are in the process of this upgrade so using node18 here would add to our TODO list on that.

Reply:
Sure, no worries. I'm not sure which is better for integrations, LATEST or 20? Up to you

Reply:
Ended up picking `NODEJS_20_X`, should make the upgrade easier since `LATEST` is still at 18 on this branch